envergo/templates/pages/geometricians.html

```diff
@@ -0,0 +1,204 @@
+{% extends 'base.html' %}
+
+{% load static l10n %}
+
+{% block title %}
+  EnvErgo : géomètres experts, planifiez vos projets d’aménagement en ne manquant aucune obligation environnementale.
+{% endblock %}
+
+{% block main-classes %}geometricians{% endblock %}
+{% block top-bar %}{% endblock %}
+
+{% block container %}
+
+  <section id="hero">
+    <div class="fr-container">
+      <div class="fr-grid-row fr-pt-4w">
+        <div class="fr-col-12 fr-col-lg-8 fr-pr-2w">
+          <h1>
+            Géomètres experts, planifiez vos projets d’aménagement en ne manquant aucune obligation
+            environnementale.
+          </h1>
+
+          <ul class="fr-badge-group fr-my-3w">
+            <li>
+              <p class="fr-badge fr-badge--blue-ecume">💧 Loi sur l'eau</p>
+            </li>
+            <li>
+              <p class="fr-badge fr-badge--blue-ecume">🌱 SAGE</p>
+            </li>
+            <li>
+              <p class="fr-badge fr-badge--blue-ecume">🐸 Natura 2000</p>
+            </li>
+            <li>
+              <p class="fr-badge fr-badge--blue-ecume">📖 Évaluation environnementale</p>
+            </li>
+          </ul>
+          <p class="fr-text fr-text--lead">
+            EnvErgo est un service public gratuit destiné aux acteurs de l'aménagement et de la construction. Avec notre
+            simulateur, découvrez les réglementations environnementales spécifiques à tout projet d'aménagement.
+          </p>
+          <a href="#simulateur"
+             class="fr-btn fr-btn--secondary fr-btn--lg fr-mb-1w"
+             data-event-category="GeometrePage"
+             data-event-action="SimulationClick"
+             data-event-name="Main">Simulez votre projet</a>
+          <br />
+          <span class="fr-text fr-text--sm">3 minutes, gratuit et anonyme</span>
+        </div>
+        <div id="geometrician-illustration" class="fr-col-12 fr-col-lg-4">
+          <img src="{% static 'images/geometrician.svg' %}"
+               alt="Illustration d'un géomètre">
+        </div>
+      </div>
+    </div>
+    <svg id="rounded-hero" viewBox="0 0 800 140">
+      <path fill="#fff" fill-opacity="1" d="M 0 100 Q 350 200 800 0 L 800 200 L 0 200 L 0 100 "></path>
+    </svg>
+
+  </section>
+
+  <section id="advantages" class="fr-container">
+    <div class="fr-grid-row fr-grid-row--gutters fr-py-5w">
+      {% for title, description, light_picture_src, dark_picture_src in advantages %}
+        <div class="fr-col fr-col-12 fr-col-md-4">
+          <img src={% static light_picture_src %} alt={{ description }} class="light-only">
```

Comment:
À noter : les images à portée purement décoratives doivent être ignorées par les technologies d'assistance en ayant une balise `alt` vide. Il me semble que c'est plutôt le cas ici.

Cf. https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#1.2

Reply:
Aussi, je crois qu'il manque des guillemets.

```suggestion
          <img src="{% static light_picture_src %}" alt="" class="light-only">
```